From fe7b9f2e175c3b179d4d227554e76380e37a3473 Mon Sep 17 00:00:00 2001
From: Jason Hung <jhung@globalscaletechnologies.com>
Date: Mon, 20 Nov 2017 09:03:52 +0800
Subject: [PATCH 3/5] git: add clocks_ddr.txt to git ignore list

add clocks_ddr.txt temporary file to git ignore list
---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 53b1d45..35a3a47 100644
--- a/.gitignore
+++ b/.gitignore
@@ -8,5 +8,6 @@ wtmi/build/*
 #ignore specific files
 ddr_static.txt
 autoconf.h
+clocks_ddr.txt
 tim/ddr/ddr_tool
 
-- 
1.9.1

